blaze/server/server.py

```diff
@@ -70,6 +85,12 @@ class Server(object):
         A dictionary mapping dataset name to any data format that blaze
         understands.
 
+    formats : ``iterable[SerializationFormat]``, optional
+        An iterable of supported serialization formats. By default, the
+        server will support JSON.
+        A serialization format is an object that supports:
+        name, loads, and dumps.
```

Comment:
No, I am not using this yet. One benefit of the namedtuple approach is that I can pass the same serialization format object to the server and the client and it will correctly generate and query the proper compute.{name} endpoint. If I passed a dict, I would need to manually keep the name in sync.